pkg/db/model.go

```diff
@@ -70,7 +70,7 @@ func ParseAccountStatus(status string) (AccountStatus, error) {
 	case "leased":
 		return Leased, nil
 	}
-	return None, fmt.Errorf("Cannot parse value %s", status)
```

Comment:
Just felt this was a more descriptions error response 
(happens if we do `GET /accounts?status=NotAValidStatus`)